### ISSUE TITLE
Provide toast options to be visible / hidden on setting Clipboard.setString('') at each event 

### ISSUE THREAD

**Issue by Shubhamrathore3196** [react-native-clipboard/clipboard#179]
## Description of Feature 

Please provide additional props of Clipboard  to show / hide toast when : 
Clipboard.setString() is triggered .

There are third party tools available that steals data saved on clipboard and to prevent this : 
our team has decided to use Clipboard.setString('') to empty string thus clearing all the data ; 

but this has triggered another issue : 
A toast is kept visible after setting Clipboard text to empty : with name : Copied to Clipboard .

The enhancement of this issue :
https://github.com/react-native-clipboard/clipboard/issues/85

has caused a new issue to reopen for our team .

Please help on this as soon as possible .

Team Airtel Bharti (India)  
shubham6.singh@airtel.com

**Comment by ravirajn22** [comment]
Hiding the default toast is impossible from Android 13. Since the toast is shown by Android when setting Clipboard data.
https://developer.android.com/develop/ui/views/touch-and-input/copy-paste#Feedback
You have to probably check if there is any possibility of clearing data instead of setting empty string. I don't know if there exist any API.

**Comment by Shubhamrathore3196** [comment]
> Hiding the default toast is impossible from Android 13. Since the toast is shown by Android when setting Clipboard data. https://developer.android.com/develop/ui/views/touch-and-input/copy-paste#Feedback You have to probably check if there is any possibility of clearing data instead of setting empty string. I don't know if there exist any API.

Yes you are right .. we have to use native textInput as last resort both for android and IOS .
